staging/src/k8s.io/apiserver/pkg/storage/etcd3/store.go

```diff
@@ -118,6 +120,11 @@ func newStore(c *clientv3.Client, quorumRead, pagingEnabled bool, codec runtime.
 	return result
 }
 
+// setLeaseReuseDurationSeconds is used for testing purpose only.
+func (s *store) setLeaseReuseDurationSeconds(d int64) {
```

Comment:
nit - no need to expose this function, call `s.leaseManager.setLeaseReuseDurationSeconds(d)` directly in the test

Reply:
fixed